client/tests/integration/queries/asset.rs

```diff
@@ -88,32 +87,6 @@ fn find_asset_total_quantity() -> Result<()> {
             .map(|account_id| <Asset as Identifiable>::Id::new(definition_id.clone(), account_id))
             .collect::<Vec<_>>();
 
-        // Give Alice ability to burn other accounts assets
-        asset_ids
-            .iter()
-            .skip(1)
-            .cloned()
-            .map(|asset_id| CanBurnUserAssets::new(asset_id).into())
-            .map(|permission_token: PermissionToken| {
-                GrantBox::new(permission_token, accounts[0].clone())
-            })
-            .zip(accounts.iter().skip(1).cloned())
-            .zip(keys.iter().cloned())
-            .map(|((grant_box, account_id), key_pair)| {
-                Transaction::new(
-                    account_id,
-                    Executable::Instructions(vec![grant_box.into()]),
-                    100_000,
-                )
-                .sign(key_pair)
-                .expect("Signing failed")
-            })
-            .for_each(|transaction| {
-                test_client
-                    .submit_transaction_blocking(transaction)
-                    .expect("Unable to execute transaction");
-            });
```

Comment:
No, because this assets are registered from `test_client` which uses `alice@wonderland` by default. That means, that `alice@wonderland` is the creator of this asset definition and has the power to burn user assets as it's allowed in validator

Reply:
is this different than before? or was this redundant even before?

Reply:
it was

Reply:
@mversic could you link to an issue here? We need to make sure that nothing slips through the cracks.

Reply:
This can be solved by #3270. It's a more global issue, but it solves this problem automatically